helm/charts/hpe-csi-driver/README.md

```diff
@@ -70,7 +70,27 @@ helm install my-hpe-csi-driver hpe-storage/hpe-csi-driver -n hpe-storage -f myva
 
 Due to the [helm limitation](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations) to not support upgrade of CRDs between different chart versions, helm chart upgrade is not supported.
 
-Our recommendation is to uninstall the existing chart and install the chart with the desired version. CRDs will be preserved between uninstall and install.
```

Comment:
We can still keep this section and add an exception for upgrade to 2.1.0?

Reply:
Yes, keep this.